pkg/api/networkservice/mechanisms/memif/constants.go

```diff
@@ -29,9 +29,9 @@ const (
 	// SocketFilename - name of the memif socketfile
 	SocketFilename = "socketfile"
 
-	// SocketFileURL - file url for the memif socketfile
-	SocketFileURL = common.InodeURL
+	// NetNSURL - url representing a inode - fmt.Sprintf("inode://%d/%d",dev,ino)
+	NetNSURL = common.InodeURL
```

Comment:
Why is it called `NetNSURL` when this is always inodeURL?

Reply:
I was using the same comments as in `kernel/{constants|helper}.go` - they are probably a little bit outdated.
Fixed comments in all places: common, kernel, memif.